hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/converter/FileIDConverter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config.converter;
+
+import com.hedera.hapi.node.base.FileID;
+import com.swirlds.config.api.converter.ConfigConverter;
+import java.util.stream.Stream;
+
+/**
+ * Config api {@link ConfigConverter} implementation for the type {@link FileID}.
+ */
+public class FileIDConverter implements ConfigConverter<FileID> {
+
+    @Override
+    public FileID convert(final String value) throws IllegalArgumentException, NullPointerException {
+        if (value == null) {
+            throw new NullPointerException("null can not be converted");
+        }
+        try {
+            final long[] nums =
```

Comment:
This is the third time I see these statements. Maybe it would be better to create a base class or a utility class, which converts a `String` into an array of `long`s.